From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Sat, 16 Jan 2021 14:30:12 -0500
Subject: [PATCH] Remove ProjectileHitEvent call when fireballs dead

The duplicate ProjectileHitEvent in EntityFireball was removed. The
event was always called before the duplicate call.

diff --git a/src/main/java/net/minecraft/world/entity/projectile/AbstractHurtingProjectile.java b/src/main/java/net/minecraft/world/entity/projectile/AbstractHurtingProjectile.java
index edc9f3450b56e0c5c12387b6fd51b9628732c372..97231f7328f0eebffcacdae5469027be8aeec3ae 100644
--- a/src/main/java/net/minecraft/world/entity/projectile/AbstractHurtingProjectile.java
+++ b/src/main/java/net/minecraft/world/entity/projectile/AbstractHurtingProjectile.java
@@ -88,7 +88,7 @@ public abstract class AbstractHurtingProjectile extends Projectile {
 
                 // CraftBukkit start - Fire ProjectileHitEvent
                 if (this.isRemoved()) {
-                    CraftEventFactory.callProjectileHitEvent(this, movingobjectposition);
+                    // CraftEventFactory.callProjectileHitEvent(this, movingobjectposition); // Paper - this is an undesired duplicate event
                 }
                 // CraftBukkit end
             }
